docs/docs/20-usage/20-pipeline-syntax.md

```diff
@@ -103,6 +103,17 @@ When using the `local` backend, the `image` entry is used to specify the shell,
 +    image: mysql
 ```
 
+When using the `lxc` backend, the `image` entry is used to specify the  `--template`  and `--release`argument to lxc-create, such as  or `debian:bullseye`, that is used to run the commands.
```

Comment:
can you just point out that the backends local, ssh and lxc does use image different and that if you need more info you should read more about that in there dedicated documentation (aka ... link to backends/X) 

and remove this specific case here?